processes/oclcClassify.py

```diff
@@ -119,7 +119,11 @@ def frbrizeRecord(self, record):
                 logger.debug(err.message)
 
     def classify_record_by_metadata_v2(self, identifier, identifier_type, author, title):
-        search_query = self.oclc_catalog_manager.generate_search_query(identifier, identifier_type, title, author)
+
+        try:
+            search_query = self.oclc_catalog_manager.generate_search_query(identifier, identifier_type, title, author)
```

Comment:
hmm are we erring out when we generate the search query?

Reply:
In main, if "generate search query" returns None because a search query can't be built, the code still sends the (empty) search query to OCLC and then fails with a warning level log and continues. I'm open to how exceptions should be handled but I do think we should avoid sending empty queries to OCLC - very flexible about how or where we do that!

Reply:
I'm going to try a different rework